autobot/telegram/parser.py

```diff
@@ -697,3 +698,595 @@ def _parse_sticker (self,key:str,val:dict) -> Sticker|StickerSet:
+        return loginurl_obj
+
+
+    def _parse_maskposition(self,key:str,val:dict) -> MaskPosition:
+        maskposition_obj = MaskPosition()
+        for k,v in val.items():
+            match k:
+                case "point"|"x_shift"|"y_shift"|"scale":
+                    setattr(maskposition_obj,k,v)
+                case _:
+                    setattr(maskposition_obj,k,v)
+        return maskposition_obj
+
+    def _parse_inlinekeyboardbutton(self,key:str,val:dict) -> InlineKeyboardButton:
+        inlinekeyboardbutton_obj = InlineKeyboardButton()
+        for k,v in val.items():
+            setattr(inlinekeyboardbutton_obj,k,v)
+        return inlinekeyboardbutton_obj
+
+    def _parse_poll(self,key:str,val:dict) -> Poll:
+        poll_obj = None
+        
+        if (key == "poll"):
+            poll_obj = Poll()
+            for k,v in val.items():
+                match k:
+                    case "options":
+                        options = []
+                        for option in v:
+                            options.append(self._parse_polloption(k,option))
+                        setattr(poll_obj,k,options)
+                    case _:
+                        setattr(poll_obj,k,v)
+        elif (key == "poll_answer"):
+            poll_obj = PollAnswer()
+            for k,v in val.items():
+                match k:
+                    case "poll_id"|"user"|"option_ids":
+                        setattr(poll_obj,k,v)
+                    case _:
+                        setattr(poll_obj,k,v)
+        elif (key == "poll_option"):
+            poll_obj = PollOption()
+            for k,v in val.items():
+                match k:
+                    case "text":
+                        setattr(poll_obj,k,v)
+                    case _:
+                        setattr(poll_obj,k,v)
+        return poll_obj
+
+    def _parse_polloption(self,key:str,val:dict) -> PollOption:
+        polloption_obj = PollOption()
+        for k,v in val.items():
+            match k:
+                case "text":
+                    setattr(polloption_obj,k,v)
+                case _:
+                    setattr(polloption_obj,k,v)
+        return polloption_obj
+
+    def _parse_document(self,key:str,val:dict) -> Document:
+        document_obj = Document()
+        for k,v in val.items():
+            match k:
+                case "thumb":
+                    thumb = self._parse_photosize(k,v)
+                    setattr(document_obj,k,thumb)
+                case _:
+                    setattr(document_obj,k,v)
+        return document_obj
+
+    def _parse_webapp(self,key:str,val:dict) -> WebAppData:
+        webapp_obj = WebAppData()
+        for k,v in val.items():
+            match k:
+                case "url"|"title"|"hide_url"|"description"|"thumb_url"|"thumb_width"|"thumb_height":
+                    setattr(webapp_obj,k,v)
+                case _:
+                    setattr(webapp_obj,k,v)
```

Comment:
The `match case` statement is not needed here. And I think this method is also responsible for parsing the rest of the objects in the webapp module